cmd/kpod/common.go

```diff
@@ -50,30 +78,59 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	return store, nil
 }
 
+func getCopyOptions(reportWriter io.Writer, signaturePolicyPath string, srcDockerRegistry, destDockerRegistry *dockerRegistryOptions, signing signingOptions) *cp.Options {
+	if srcDockerRegistry == nil {
+		srcDockerRegistry = &dockerRegistryOptions{}
+	}
+	if destDockerRegistry == nil {
+		destDockerRegistry = &dockerRegistryOptions{}
+	}
+	srcContext := srcDockerRegistry.getSystemContext(signaturePolicyPath)
+	destContext := destDockerRegistry.getSystemContext(signaturePolicyPath)
+	return &cp.Options{
+		RemoveSignatures: signing.RemoveSignatures,
+		SignBy:           signing.SignBy,
+		ReportWriter:     reportWriter,
+		SourceCtx:        srcContext,
+		DestinationCtx:   destContext,
+	}
+}
+
+func getPolicyContext() (*signature.PolicyContext, error) {
+	var systemContext types.SystemContext
+	policy, err := signature.DefaultPolicy(&systemContext)
```

Comment:
_This_ is where `types.SystemContext.SignaturePolicyPath` is used. So if the code is passing an empty `SystemContext` here, all the other places setting `.SignaturePolicyPath` are irrelevant.